NicholasSaravia/directory
# chore: Add Material UI

### Description

this chore adds material ui following the [material-ui installation guide](https://mui.com/material-ui/getting-started/installation/)

### How To Test

checkout branch and use material ui components on `pages/index.tsx`
all base material components should work 

### Pics / Vids

n/a

### Libraries Added

base libraries
- @mui/material @emotion/react @emotion/styled 

base fonts
- @fontsource/roboto

icons
- @mui/icons-material
